projects/1.20/assets/sons-of-sins/sons_of_sins/lang/zh_cn.json

```
@@ -0,0 +1,210 @@
+{
+  "advancements.a_heal_stick.descr": "使用装满骨灰的以太剑",
+  "advancements.a_heal_stick.title": "治愈法杖",
+  "advancements.ash_diamond.descr": "在血液中投掷16个灰烬",
```

```suggestion
  "advancements.ash_diamond.descr": "向血液中投掷灰烬",
```
原文`Throw ashes in blood`
是否要求玩家位于血液中再投掷灰烬？还是玩家在血液内/外，向血液中投掷灰烬？
`16个`是哪里来的？看key名像是进度描述，应该不需要技术性的具体解释；
如果防止玩家“只投一个”的话，可以用不定量描述词，如`一些`、`大量`